chaoskube/chaoskube.go

```diff
@@ -7,6 +7,8 @@ import (
 
 	log "github.com/sirupsen/logrus"
 
+	"strings"
```

Comment:
Stdlib import live in the very first section, i.e. next to `errors`, `fmt`, etc.